src/ui/vite.config.custom.ts

```diff
@@ -29,11 +31,11 @@ export default defineConfig({
 			},
 		},
 		rollupOptions: {
-			external: ["vue", "../injectionKeys"],
+			external: ["vue", injectionKeys],
```

Comment:
I have to use this code to make the build of custom component working on windows.

```javascript
rollupOptions: {
			external: ["vue", "@/injectionKeys"],
			output: {
				globals: {
					vue: "vue",
					[path.resolve("src/injectionKeys")]: "injectionKeys",
				},
			},
		},
		```

Reply:
I implemented it in https://github.com/writer/writer-framework/pull/718/commits/d6fb5ec06ce879305bc244e0fc5919691502968b